ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/MetricsInterceptorSpec.scala

```diff
@@ -60,6 +66,23 @@ final class MetricsInterceptorSpec
     }
   }
 
+  it should "count the gRPC return status" in {
+    val metrics = new Metrics(new MetricRegistry)
+    serverWithMetrics(metrics, new HelloServiceAkkaImplementation).use { channel: Channel =>
+      for {
+        _ <- HelloServiceGrpc.stub(channel).single(HelloRequest(0))
+        _ <- HelloServiceGrpc.stub(channel).fails(HelloRequest(1)).failed
+      } yield {
+        val okCounter = metrics.daml.lapi.return_status.forCode(Status.Code.OK.toString)
```

Comment:
I would use "200" in the test instead of `Status.Code.OK.toString` as by reading this I am not sure you get a right value out :)

Reply:
Good call, `metrics.daml.lapi.return_status.forCode("OK")` would be better.